spec/index.bs

```diff
@@ -163,7 +163,10 @@ enum GPUTextureFormat {
     "rgba32float",
     /* Depth and Stencil formats */
     "depth32float",
-    "depth32float-stencil8"
+    // depth24plus has a precision of 1 ULP <= 1/(2**24).
+    // (Unlike depth24unorm, which has a precision of 1 ULP = 1/(2**24-1).)
```

Comment:
it's a bit strange to refer to non-existent "depth24unorm" here

Reply:
How about referring generically to "typical 24-bit unsigned normalized formats" or specifically to an underlying API, e.g. Vulkan D24_UNORM?

Reply:
generic description sounds good to me

Reply:
Done